app/presenters/organisations/show_presenter.rb

```diff
@@ -0,0 +1,59 @@
+module Organisations
+  class ShowPresenter
+    attr_reader :org
+
+    def initialize(organisation)
+      @org = organisation
+    end
+
+    def prefixed_title
+      prefix = needs_definite_article?(org.title) ? "the " : ""
+      (prefix + org.title)
+    end
+
+    def latest
```

Comment:
Would it make sense to have a 'News' or 'Documents' presenter, which this could then live in along with the featured news stuff. As we're splitting things into different presenters according to use, I'm not sure it makes sense for this to live here.

Reply:
If you do choose to keep it here though, it should have a name like `latest_documents` so it's clear what this actually does.